services/web-app/pages/libraries/[host]/[org]/[repo]/[library]/[ref]/design-kits.js

```diff
@@ -4,13 +4,26 @@
  * This source code is licensed under the Apache-2.0 license found in the
  * LICENSE file in the root directory of this source tree.
  */
-import { Column, Grid } from '@carbon/react'
+
+import { Column, Grid, Link as CarbonLink, Tag } from '@carbon/react'
+import { RulerAlt } from '@carbon/react/icons'
+import { AnchorLink, AnchorLinks } from '@carbon-platform/mdx-components'
+import { get, groupBy } from 'lodash'
```

Comment:
can we use the new form of import here? I think it's this
```suggestion
import get from 'lodash/get'
import groupBy from 'lodash/groupBy'
```